posthog/models/historical_version.py

```diff
@@ -0,0 +1,42 @@
+from django.db import models
+from django.utils import timezone
+
+
+class HistoricalVersion(models.Model):
+    """
+    We don't _only_ store foreign references cos the referenced model might get deleted.
+    The history log should be relatively immutable
+
+    Everything in the log must have either a team id or an organization id
+    """
+
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["team_id", "name", "versioned_at"], name="unique_version"),
```

Comment:
Shouldn't `organization_id` also go in the list? Not sure how to read this line in the context of "Everything in the log must have either a team id or an organization id". What will happen if a few organizations save something with the same `name` at the same time?